### ISSUE TITLE
Feature request: Media controls for music

### ISSUE THREAD

**Issue by FoobarTheFatPenguin** [shlomif/PySolFC#343]
I have come to like the background music feature. Sometimes, however, I wish I could repeat a track. At other times, the playback seems stuck in a loop (one song repeating over and over and over - haven't found a way to reproduce yet). In those cases, I wish I could just skip to the next track.

I think it would be cool if users could add some simple media controls to the toolbar (play/pause, stop, next, previous). Displaying the name of the file that is currently playing in the status bar would also be helpful.

Just a suggestion. I understand this is not part of the core functionality of the program and thus probably not of the highest priority.

**Comment by joeraz** [comment]
You use to be able to use Control+U to advance to the next song - I think that was broken in some of the recent releases, but I've already fixed that for the next one.

My issue with putting it in the regular toolbar is that I feel some of the options could be confusing - it might be tough to distinguish between "pause game" and "pause song" for example.  I could see about making a separate toolbar, or even a music player window similar to the find card dialog.  Will need to think this one over.

**Comment by FoobarTheFatPenguin** [comment]
How about making it clear via the icon, e.g. by adding a music note next to the play/pause/stop/next/prev symbols.

Quick example sketch:
![mediacontrols](https://github.com/shlomif/PySolFC/assets/96542877/188a147e-8739-49fd-b96a-f2d98b4e34b2)

**Comment by joeraz** [comment]
Yes, but if we have to create unique icons, we'd need to create them for each size and toolbar style.

**Comment by hajo4** [comment]
I'd like to see the title of the currently playing song, eg. somewhere on the statusbar.
And a hotkey to toggle music on/off, without having to go thru options.
